<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>559</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>758</width>
    <height>559</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>758</width>
    <height>559</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>- ACTUALIZAR -</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEditActualizaDomicilio">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhUppercaseOnly</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditActualizaLocalidad">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>160</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhUppercaseOnly</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditActualizaCelular">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>190</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelActualizaCelular">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>CELULAR</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelActualizaTelLinea">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>190</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TEL. LINEA</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditActulizaNombre">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhPreferUppercase|Qt::ImhUppercaseOnly</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditActualizaTelLinea">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>190</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelActualizaEmail">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>E-MAIL</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelActualizaApellido">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>APELLIDO</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelActualizaNombre">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>NOMBRE</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelActualizaRedSocial">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>RED SOCIAL</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditActualizaApellido">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhUppercaseOnly</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditActualizaDNI">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelActualizaLocalidad">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>160</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>LOCALIDAD</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditActualizaRedSocial">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>250</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelActualizaDomicilio">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>DOMICILIO</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditActualizaEmail">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>220</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhEmailCharactersOnly</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelActualizaDNI">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>DNI</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditActualizaObservacion">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>280</y>
      <width>651</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelActualizaObservacion">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>OBSERVACIÓN</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>370</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>DATOS ACTUALIZADOS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnActualizaSalir">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>480</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>SALIR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnActualizaSocio">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>310</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>ACTUALIZAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnActualizaVolver">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>IR a CONSULTA</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelActuSocioNum">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>SOCIO N° </string>
    </property>
   </widget>
   <widget class="QLabel" name="labelActualizaMostrarSocio">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidgetActualizaSocio">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>741</width>
      <height>111</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>741</width>
      <height>111</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <column>
     <property name="text">
      <string>Socio</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Apellido</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Nombre</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>DNI</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Domicilio</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Localidad</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Celular</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Tel. linea</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>E-Mail</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Red Social</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Observación</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Cuota</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Fecha Alta</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Fecha Pago</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Fecha Vence</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Sexo</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Actividad</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Promo</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="labelActuaImagenLogo">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>30</y>
      <width>131</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/imagenes/logo_orbis.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>logo_orbis.jpg</pixmap>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="imagenes.qrc"/>
 </resources>
 <connections/>
</ui>
